include/zephyr/net_buf.h

```diff
@@ -1098,6 +1098,9 @@ struct net_buf_pool {
 	/** Total size of the pool. */
 	const uint16_t pool_size;
 
+	/** Maximum count of used buffers. */
+	uint16_t max_used;
```

Comment:
One thing I started wondering: perhaps this member should also be behind `#if defined(CONFIG_NET_BUF_POOL_USAGE)`?

Reply:
It is (open some more context above)

Reply:
Right. Thanks.